### PR TITLE
Clean up HTML

### PR DESCRIPTION
**Pull request by mathiasbynens**
See inline comments for details.

### CODE REVIEW

File: ssl-config-generator/index.html [hunk 1]
```diff
@@ -4,8 +4,8 @@
 <head>
     <meta charset="utf-8" />
     <title>Generate Mozilla Security Recommended Web Server Configuration Files</title>
-    <link rel="stylesheet" type="text/css" media="all" href="../css/screen.css" />
```
Comment (mathiasbynens) [inline ssl-config-generator/index.html:7]: Unneeded `type="text/css"`.

File: ssl-config-generator/index.html [hunk 2]
```diff
@@ -4,8 +4,8 @@
...
-    <link href="//mozorg.cdn.mozilla.net/media/css/tabzilla-min.css" rel="stylesheet" />
+    <link rel="stylesheet" href="../css/screen.css" />
+    <link rel="stylesheet" href="https://mozorg.cdn.mozilla.net/media/css/tabzilla-min.css" />
```
Comment (mathiasbynens) [inline ssl-config-generator/index.html:8]: Forcing HTTPS as per https://github.com/konklone/cdns-to-https#background.

File: ssl-config-generator/index.html [hunk 3]
```diff
@@ -39,12 +39,12 @@
     </script>
     <script>
         if (window.location.protocol != "https:")
-            window.location.href = "https:" + window.location.href.substring(window.location.protocol.length);
+            window.protocol = "https:";
```
Comment (mathiasbynens) [inline ssl-config-generator/index.html]: Much simpler, and does the same thing.

File: ssl-config-generator/index.html [hunk 4]
```diff
@@ -26,11 +26,11 @@
...
-    <script src="//ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
-    <script src="//cdnjs.cloudflare.com/ajax/libs/handlebars.js/2.0.0/handlebars.min.js"></script>
-    <script src="//mozorg.cdn.mozilla.net/en-US/tabzilla/tabzilla.js"></script>
-    <link rel="stylesheet" href="//ajax.googleapis.com/ajax/libs/jqueryui/1.11.3/themes/smoothness/jquery-ui.css">
-    <script src="//ajax.googleapis.com/ajax/libs/jqueryui/1.11.3/jquery-ui.min.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/handlebars.js/2.0.0/handlebars.min.js"></script>
+    <script src="https://mozorg.cdn.mozilla.net/en-US/tabzilla/tabzilla.js"></script>
+    <link rel="stylesheet" href="https://ajax.googleapis.com/ajax/libs/jqueryui/1.11.3/themes/smoothness/jquery-ui.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.11.3/jquery-ui.min.js"></script>
```
Comment (jrchamp) [inline ssl-config-generator/index.html]: Consider bumping to the current version while you're modifying these lines.

Comment (mathiasbynens) [inline ssl-config-generator/index.html]: Are you sure that won’t break anything? Seems like that deserves its own commit / PR, IMHO.

File: ssl-config-generator/index.html (hunk 3 above)
Comment (jrchamp) [inline ssl-config-generator/index.html]: The `if` checks `window.location.protocol`, but the new line modifies `window.protocol`; maybe these should be the same variable?

Comment (mathiasbynens) [inline ssl-config-generator/index.html]: Whoops, yes it should be `location.protocol`. My bad. Fixing…

File: ssl-config-generator/index.html [hunk 5]
```diff
@@ -632,8 +629,8 @@
     <script>
         (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
         (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-        m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-        })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+        m=s.getElementsByTagName(o)[0];a.src=g;m.parentNode.insertBefore(a,m)
+        })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
```
Comment (jrchamp) [inline ssl-config-generator/index.html:633]: Why remove `async`? It's not preferable to have the analytics.js delay the page load.

File: ssl-config-generator/index.html (hunk 4 above)
Comment (mathiasbynens) [inline ssl-config-generator/index.html]: Added a separate commit to this PR.

File: ssl-config-generator/index.html (hunk 5 above)
Comment (mathiasbynens) [inline ssl-config-generator/index.html]: I agree, but script-inserted scripts delay page load already. `scriptElement.async = true` [literally does nothing except in Firefox 3.6](https://mathiasbynens.be/notes/async-analytics-snippet#async).

Comment (mathiasbynens) [inline ssl-config-generator/index.html:633]: I agree, but script-inserted scripts are asynchronous already. `scriptElement.async = true` [literally does nothing except in Firefox 3.6 and OmniWeb 622](https://mathiasbynens.be/notes/async-analytics-snippet#async).

**Comment by mathiasbynens** [comment]
Thanks for the review @jrchamp! :+1: I’ve updated the patch.

File: ssl-config-generator/index.html (hunk 5 above)
Comment (jrchamp) [inline ssl-config-generator/index.html:633]: Ah! Good to know.

Any idea why Google has not implemented this change upstream?

Comment (mathiasbynens) [inline ssl-config-generator/index.html:633]: No clue. Maybe they care about performance of their tracking code in those outdated/obscure browsers, but I somehow doubt that’s the reason.

**Comment by jvehent** [comment]
Could you rebase this so it can be merged? Thanks.

**Comment by jvehent** [comment]
Ping. Any chance to get this patch rebased?

**Comment by jvehent** [comment]
merged in #173

File: ssl-config-generator/index.html (hunk 5 above)
Comment (april) [reply ssl-config-generator/index.html:633]: I suspect they're just terrified of tweaking anything with GA.  :)
